tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+# role tasks
+- name: Verify chronyd is installed
+  package:
+    name: "{{ chrony_package }}"
+    state: present
+  become: true
+  become_user: "{{ chrony_become_user }}"
```

Comment:
Put all of these tasks in a block with a single set of `become` options to cut down on repetition

Reply:
Personally I prefer this syntax for become/become_user. I like to see it explicitly listed on every task so that I know what is and isn't using root privileges. I wish I could count the number of times I've started adding `become` to tasks only to later find out it was on the `include_tasks` statement in the parent playbook or on a block at the end of the file.

Reply:
No task inclusion is happening here. In my opinion the block `become` is just as explicit as doing it per-task; in this case the difference is cosmetic.

Reply:
+1 for blocks.  I think it may be a little less error prone than having to add "become" per-task.  If a task is indented in a block, it's pretty clear to me that behavior has changed in some way beyond the default.